catroid/src/main/java/org/catrobat/catroid/ui/MainMenuActivity.java

```diff
@@ -96,6 +99,9 @@
 	public static final String SHARED_PREFERENCES_SHOW_BROWSER_WARNING = "shared_preferences_browser_warning";
 	public static final int REQUEST_CODE_GOOGLE_PLUS_SIGNIN = 100;
 
+	public static final String RESTART_INTENT = "restart";
+	private final int restartDelayAfterAppClose = 500;
```

Comment:
please specify the unit in the variables name.
i guess its milliseconds so it would be restartDelayMsAfterAppClose